packages/graphback/src/graphback.ts

```diff
@@ -0,0 +1,48 @@
+import { resolve } from 'path';
+import { loadConfigSync } from 'graphql-config';
+import { 
+  CRUDService,
+  GraphbackDataProvider,
+  createRuntimeServices,
+} from '@graphback/runtime';
+import { GraphbackRuntime } from './GraphbackRuntime';
+import { PubSubEngine } from 'graphql-subscriptions';
+
+export interface GraphbackOptions<Provider extends GraphbackDataProvider, Service extends CRUDService> {
+  db: any
+  pubSub: PubSubEngine,
+  provider: new (...args: any[]) => Provider,
+  service: new (...args: any[]) => Service;
+}
+
+/**
+ * Create a Graphback Runtime
+ * 
+ * @param options 
+ */
+export function graphback<Provider extends GraphbackDataProvider, Service extends CRUDService>(options: GraphbackOptions<Provider, Service>) {
+  const { db, pubSub, provider, service } = options
+
+  const projectConfig = loadConfigSync({
```

Comment:
There was a separate discussion previously where the runtime templates will no longer use `graphql-config`, as it is mainly used for code generation, which is not a factor in a runtime Graphback API.

So eventually we can drop support for graphql-config completely, and the user can provide a JSON configuration object in `options`.

https://github.com/aerogear/graphback/issues/1146

Reply:
Hmm.. yaml will be used for generate but if we drop resolves generation we will simply have schema plugin so for runtime use cases having ability to define plugins in the code with defaults will be much better than using config - while config can still stay I do not need to be used by default